.travis.yml

```diff
@@ -8,4 +8,11 @@ before_install:
   - rm -r main
 
 before_script:
+  # golint
+  - go get -u github.com/golang/lint/golint
+  # peg
+  - go get -u github.com/pointlander/peg
   - cqlsh -f schema/schema_test.cql
+
+script:
+  - ./build_verify.sh
```

Comment:
Should explicitly add `go test -v ./...` here, since otherwise travis assumes that script has everything we want to run in the build.